src/main/kotlin/org/rust/ide/refactoring/changeSignature/RsChangeSignatureUsageProcessor.kt

```diff
@@ -132,11 +134,14 @@ private fun findNameConflicts(
         if (item == function) continue
         if (!item.isEnabledByCfgSelf) continue
 
-        if (item.name == config.name) {
+        val namedItem = item as? RsNamedElement ?: continue
```

Comment:
```suggestion
        if (item !is RsNamedElement) continue
```

Reply:
Actually this is needed, otherwise the `name` method call will be ambiguous.
![image](https://user-images.githubusercontent.com/4539057/114436919-33cf8500-9bc6-11eb-9897-b52abe6c3943.png)